scripts/test/integration-sqlite-setup.sh

```diff
@@ -1,4 +1,4 @@
 pip uninstall -y dbt-core dbt-sqlite openlineage-airflow openlineage-integration-common; \
 rm -rf airflow.*; \
 airflow db init; \
-pip install 'dbt-core==1.4' 'dbt-sqlite<=1.4' 'dbt-databricks<=1.4' 'dbt-postgres<=1.4'
+pip install 'dbt-core==1.8' 'dbt-sqlite<=1.8' 'dbt-databricks<=1.8' 'dbt-postgres<=1.8'
```

Comment:
```suggestion
pip install 'dbt-core==1.5' 'dbt-sqlite<=1.5' 'dbt-databricks<=1.5' 'dbt-postgres<=1.5'
```

Reply:
Latest dbt-sqlite adaptor is 1.5 https://pypi.org/project/dbt-sqlite/#history

Reply:
yes, we specify `dbt-sqlite<=1.8`, so whenever a newer version is released it would be picked up & should ideally be still compatible with `dbt-core==1.8`. This is my rationale. WDYT?